### ISSUE TITLE
SCREAMv0 ne256 and ne1024 failed in initialization on cori knl

### ISSUE THREAD

**Issue by jli628** [E3SM-Project/scream#1798]
I tried to test SCREAM (master branch downloaded about two weeks ago) on cori using knl. The model can run successfully with resolution=ne30pg2_r0125_oRRS18to6v3 and compset=F2010-SCREAM-HR-DYAMOND2. However, when I change the resolution to ne256pg2_r0125_oRRS18to6v3 and ne1024pg2_r0125_oRRS18to6v3 (the number of nodes will increase to 512), the model crashed during the initialization.

I am attaching the log files and the run script below. Does anyone encounter the same problem before? Any suggestions? Thanks.

[atm.log.ne256.docx](https://github.com/E3SM-Project/scream/files/9146226/atm.log.ne256.docx)
[atm.log.ne1024.docx](https://github.com/E3SM-Project/scream/files/9146227/atm.log.ne1024.docx)
[cpl.log.ne256.docx](https://github.com/E3SM-Project/scream/files/9146228/cpl.log.ne256.docx)
[cpl.log.ne1024.docx](https://github.com/E3SM-Project/scream/files/9146229/cpl.log.ne1024.docx)
[e3sm.log.ne256.docx](https://github.com/E3SM-Project/scream/files/9146232/e3sm.log.ne256.docx)
[e3sm.log.ne1024.docx](https://github.com/E3SM-Project/scream/files/9146233/e3sm.log.ne1024.docx)
[run.production.ne30pg2_scream.dyamond2_HR.docx](https://github.com/E3SM-Project/scream/files/9146234/run.production.ne30pg2_scream.dyamond2_HR.docx)

**Comment by jli628** [comment]
Please delete the file suffix to read the files. The suffix is sh for the run script. I have to change the suffix to upload the files. Sorry for any inconvenience.

**Comment by PeterCaldwell** [comment]
Have you read https://acme-climate.atlassian.net/wiki/spaces/NGDNA/pages/3386015745/How+To+Run+SCREAMv1? It says that only ne256np4_oRRS18to6v3 has been tested at ne256. Nobody has been able to run anything at ne1024 yet - we just created our first initial condition for ne1024 a few days ago.

**Comment by PeterCaldwell** [comment]
I posted a comment, but realized it was for the C++ code so deleted it.

**Comment by PeterCaldwell** [comment]
I'm pretty sure the issue is that your requested walltime is 30 min. High-res simulations can't complete initialization during that time. e3sm.log supports this - the error message is: 

>srun: error: nid02583: tasks 1,3-7: Killed
srun: launch/slurm: _step_signal: Terminating StepId=61190710.0
 33: forrtl: error (78): process killed (SIGTERM)
  0: slurmstepd: error: *** STEP 61190710.0 ON nid02583 CANCELLED AT 2022-07-19T14:00:49 ***
 93: forrtl: error (78): process killed (SIGTERM)

sigterm means the run timed out.

**Comment by jli628** [comment]
Thanks, Caldwell. After I commented out "source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_cori-knl.sh", it seems ne256 works. I will try ne1024 again when cori is back.

**Comment by jli628** [comment]
@PeterCaldwell 
I tried ne1024 on cori-knl using 512 nodes. It crashed with out-of-memory, as shown below:

  0:  Reading setup_nml
  0:  Reading grid_nml
  0:  Reading ice_nml
  0:  Reading tracer_nml
  0: CalcWorkPerBlock: Total blocks:        575 Ice blocks:        575 IceFree blocks:          0 Land blocks:          0
  0: MCT::m_Router::initp_: GSMap indices not increasing...Will correct
  0: MCT::m_Router::initp_: RGSMap indices not increasing...Will correct
  0: MCT::m_Router::initp_: RGSMap indices not increasing...Will correct
  0: MCT::m_Router::initp_: GSMap indices not increasing...Will correct
253: slurmstepd: error: Detected 1 oom-kill event(s) in StepId=61256624.0. Some of your processes may have been killed by the cgroup out-of-memory handler.
 89: slurmstepd: error: Detected 1 oom-kill event(s) in StepId=61256624.0. Some of your processes may have been killed by the cgroup out-of-memory handler.

I found you used 1536 nodes for your F2010-SCREAM-HR-DYAMOND2 simulation. Were you able to run it successfully with fewer nodes? Thanks.
